plugins/grid-bookmark/src/GridBookmarkWidget/components/BookmarkGrid.tsx

```diff
@@ -66,98 +66,97 @@ const BookmarkGrid = observer(function ({
-              >
-                {value}
-              </Link>
-            ),
-          },
-          {
-            field: 'label',
-            headerName: 'Label',
-            width:
-              bookmarkRows.length > 0
-                ? measureGridWidth(bookmarkRows.map(row => row.label))
-                : measureText('label'),
-            editable: true,
-          },
-          ...(selectedAssemblies.length === validAssemblies.size
-            ? [
-                {
-                  field: 'assemblyName',
-                  headerName: 'Assembly',
-                  width:
-                    bookmarkRows.length > 0
-                      ? measureGridWidth(
-                          bookmarkRows.map(row => row.assemblyName),
-                        )
-                      : measureText('assembly'),
-                },
-              ]
-            : []),
-        ]}
-        onCellDoubleClick={({ row }) => {
-          setDialogOpen(true)
-          setDialogRow(row)
-        }}
-        processRowUpdate={row => {
-          const target = bookmarkRows[row.id]
-          model.updateBookmarkLabel(target, row.label)
-          return row
-        }}
-        onProcessRowUpdateError={e => session.notify(e.message, 'error')}
-        checkboxSelection
-        onRowSelectionModelChange={newRowSelectionModel => {
-          if (bookmarksWithValidAssemblies.length > 0) {
-            model.setSelectedBookmarks(
-              newRowSelectionModel.map(value => ({
-                ...bookmarkRows[value as number],
-              })),
-            )
-            setRowSelectionModel(newRowSelectionModel)
-          }
-        }}
-        rowSelectionModel={rowSelectionModel}
-        disableRowSelectionOnClick
-      />
-      {dialogOpen ? (
+      <div ref={ref}>
+        <ResizeBar
+          widths={widths}
+          setWidths={setWidths}
+          scrollLeft={scrollLeft}
+        />
+        <DataGrid
+          autoHeight
+          density="compact"
+          rows={rows}
+          columns={[
+            {
+              ...GRID_CHECKBOX_SELECTION_COL_DEF,
+              width: widths[0],
+            },
+            {
+              field: 'locString',
+              headerName: 'Bookmark link',
+              width: widths[1],
+              renderCell: ({ value, row }) => (
+                <Link
+                  className={classes.link}
+                  href="#"
+                  onClick={async event => {
+                    event.preventDefault()
+                    const { views } = session
+                    await navToBookmark(value, row.assemblyName, views, model)
+                  }}
+                >
+                  {value}
+                </Link>
+              ),
+            },
+            {
+              field: 'label',
+              headerName: 'Label',
+              width: widths[2],
+              editable: true,
+            },
+            {
+              field: 'assemblyName',
```

Comment:
assembly name column is no longer omitted when only one assembly is selected -- is this intentional?

Reply:
ya, I just added it to show the assembly name column always now. it seemed not that important to hide it conditionally, and complicated the code a bit (was a bit hard to integrate with the resize bar. do-able, but didn't seem like necessary)